tequila_fab/__init__.py

```diff
@@ -74,6 +74,8 @@ def deploy(play=None, extra_vars=None, branch=None, limit=None):
         cmd.append("-e repo_branch=%s" % branch)
     if limit:
         cmd.append("-l %s" % limit)
+    if verbose:
+        cmd.append("-%s" % verbose)
```

Comment:
I'm wondering if we should do a little validation to make sure that `verbose` is a string containing only the letter `v` (`set(verbose) == {'v'}`). If it doesn't, I'm not sure what the better option is.... 
1) ignoring the parameter, or
2) setting verbose to `vvv`, assuming that the user wants more verbosity

Reply:
Good idea, added!